core/interface.go

```diff
@@ -0,0 +1,16 @@
+package core
+
+// AppStatusHandler interface will handle different implementations of monitoring tools, such as Prometheus of term-ui
+type AppStatusHandler interface {
+	Increment(key string)
+	Decrement(key string)
+	SetInt64Value(key string, value int64)
+	SetUInt64Value(key string, value uint64)
+	GetValue(key string) float64
```

Comment:
Should not have/implemented this func. Those kind of object are "output" structs. Not "input/output"

Reply:
removed